app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/164372290-b7fbec90-c076-410b-9d00-03522fcc5bbf.png)

Your log in view does not look like the wireframe:

Your Implementation | Wireframe
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/164372366-45bd849e-eb0a-457b-be0f-527185b46561.png) | ![image](https://user-images.githubusercontent.com/57421823/164372398-0c9ace44-ced3-489a-9c65-cd854b1a1231.png)

Therefore:

- [ ] Please, kindly make your styling look more like the wireframe in order to meet this particular requirement.

For instructions, refer to the link in the project requirements 💪